correlation_rules/aws_potentially_compromised_service_role.yml

```diff
@@ -22,9 +22,9 @@ Detection:
         Match:
           - On: requestParameters.roleArn
     Schedule:
-      RateMinutes: 60
+      RateMinutes: 360
```

Comment:
This was already at 60, and I wasn't sure if should downgrade the `LookbackWindowMinutes` to 90 or update the `RateMinutes` to something like `720` so I kind of met in the middle.

Reply:
That seems fine!

Reply:
Lookback shouldn't need to be larger than how late data arrives. for Cloudtrail I would 60 is more than enough?

Reply:
err 90

Reply:
so like `RateMinutes: 60`  `LookbackWindowMinutes: 90` like the others?